irods_specific_implementation/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
-cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
+cmake_minimum_required(VERSION 3.11.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
```

Comment:
do we need 3.11.0 actually minimum required?

Reply:
i was getting a cmake error which lead this. this change isn't needed though. i'll revert it or use something in between.